Source/Fuse.Controls.TimePicker/TimePicker.uno

```diff
@@ -100,9 +96,19 @@ namespace Fuse.Controls
 			get { return (ITimePickerView)NativeView; }
 		}
 
-		void ITimePickerHost.OnValueChanged()
+		protected override void PushPropertiesToNativeView()
+		{
+			var tpv = TimePickerView;
+			if (tpv != null)
+			{
+				tpv.Value = Value;
+				tpv.Is24HourView = Is24HourView;
+			}
+		}
+
+		void ITimePickerHost.OnValueChanged(DateTime newValue)
 		{
-			OnValueChanged(this);
+			SetValue(newValue, null);
```

Comment:
The `null` looks wrong, it should be the view I think. There's a condition in `SetValue` that checks the origin to prevent sending the message back-there.

Reply:
Yeah I thought that a bit strange too, but I wrote it following this line in `RangeControl.uno`: https://github.com/fusetools/fuselibs-public/blob/master/Source/Fuse.Controls.Primitives/RangeControls/RangeControl.uno#L246 . Perhaps they should both be changed?

Reply:
Yeah, it looks like that is wrong as well. We still have lots of inconsistencies with value handling in our controls unfortunately.

Reply:
Gotcha. I'll go ahead and make a PR for that specific case too then.

Reply:
https://github.com/fusetools/fuselibs-public/pull/778